src/components/SceneConfigMenu.js

```diff
@@ -398,6 +399,7 @@ class ConfigModal extends Component {
       <div>
         {!isDisabled ?
         <div>
+          <Tooltip title = "Scene Config Menu">
```

Comment:
Although this is what we refer to it as internally, it is referred to officially as either "Scene Settings" or "Scene Configuration".